Docs: Lifecycle Management of Counters and Lists in REST (#3560)
googleforgames/agones: site/content/en/docs/Guides/Client SDKs/local.md

{"message":"gameserver update received","severity":"info","time":"2019-10-30T21:46:18.179459+03:00"}
```

> {{% feature publishVersion="1.38.0" %}}

Reviewer: This doesn't need to be feature gated - it's applicable everywhere., otherwise, this looks great!

> {{% feature publishVersion="1.38.0" %}}

Reviewer 2: ```suggestion
```

Author: 🤔I have removed the feature shortcode in the [latest commit](https://github.com/googleforgames/agones/pull/3560/commits/561f1e4e693fa79a966e4f675311daadc08dadbf_

Reviewer 2: Ah yep, looks like you added the new commit before I finished my review. Looks good!

Commandline flags (e.g. `--local`) are exactly the same as command line flags when utilising a pre-built binary.

> **Enabling Specific Features:**
> For development and testing purposes, you might want to enable specific features in the SDK Server. To do this, set the FEATURE_GATES environment variable before running the server. For instance:
> ```bash
> FEATURE_GATES="FeatureName=true" go run cmd/sdk-server/main.go --local
> ```
> 
> Replace `FeatureName` with the name of the feature you want to enable. This command starts the local SDK Server with the specified feature activated.

Reviewer: Let's create move this right after the section "Running In “Local Mode".

````suggestion

## Enabling Feature Gates

For development and testing purposes, you might want to enable specific [features gates]({{< relref "../feature-stages.md#feature-gates" >}}) in the local SDK Server. 

To do this, you can either set the `FEATURE_GATES` environment variable or use the `--feature-gates` command line parameter like so, with the same format as utilised when [configuring it on a Helm install]({{< ref "/docs/Installation/Install Agones/helm.md#configuration" >}}).

For example:

```bash
./sdk-server.linux.amd64 --local --feature-gates Example=true
```
or 
```bash
FEATURE_GATES=Example=true ./sdk-server.linux.amd64 --local
```

````

You'll probably have to fix my link paths though - I did this off the top of my head.

Author: Moved `Enabling Feature Gates` section to immediately follow the `Running In “Local Mode"`

> **Setting the HTTP port:**
> To customize the SDK Server, set the HTTP port to 9358, as used by the local server, by running this command in your terminal: `export AGONES_SDK_HTTP_PORT=9358`

Reviewer: ```suggestion
```

This isn't accurate for the local SDK server, it's either --http-port argument, or HTTP_PORT (similar for grpc). Since port changing isn't that common, I'd suggest not worrying about documenting this -- unless anyone feels strongly otherwise?

If people really want it, they can run `--help` on the local server as well to get all the command line params.

Author: I've eliminated this `Setting the HTTP port:` section
